README.md

```diff
@@ -445,6 +445,7 @@ Tells Conan where to install the package. It is similar to specifying `CMAKE_INS
 The following `options` are available to pass on `conan install` when building the oneMKL library:
 
 - `build_shared_libs=[True | False]`. Setting it to `True` enables the building of dynamic libraries. The default value is `True`.
+- `target_domains=[<any value>]`. Setting it to `blas` or any other domain, enables building of that specific domain only. If not defined, the default value is all supported domains.
```

Comment:
Do you mean it should be list of values?
```suggestion
- `target_domains=[<list of values>]`. Setting it to `blas` or any other domain, enables building of that specific domain only. If not defined, the default value is all supported domains.
```

Reply:
if conan supports list, then yes, <list of values>
Maybe something like this:
Setting it to `blas` or any list of supported domains, enables building of that(those) specific domain(s) only.

Reply:
I updated the description. Let me know if there is anything else to be changed.

Reply:
Thanks, looks good to me.